pkgs/development/python-modules/pysvn/default.nix

```
@@ -8,52 +9,70 @@
 
 buildPythonPackage rec {
   pname = "pysvn";
-  version = "1.8.0";
-  disabled = isPy3k;
+  version = "1.9.12";
   format = "other";
 
   src = fetchurl {
     url = "http://pysvn.barrys-emacs.org/source_kits/${pname}-${version}.tar.gz";
-    sha256 = "0srjr2qgxfs69p65d9vvdib2lc142x10w8afbbdrqs7dhi46yn9r";
+    sha256 = "sRPa4wNyjDmGdF1gTOgLS0pnrdyZwkkH4/9UCdh/R9Q=";
   };
 
-  buildInputs = [ pkgs.subversion pkgs.apr pkgs.aprutil pkgs.expat pkgs.neon pkgs.openssl ]
-    ++ (if stdenv.isLinux then [pkgs.e2fsprogs] else []);
+  buildInputs = with pkgs; [ bash subversion apr aprutil expat neon openssl ]
```

generally i'ts preferred not to import from an entire package set. Makes it much more difficult to override packagse
```suggestion
  buildInputs = [ bash subversion apr aprutil expat neon openssl ]
```